src/workerd/jsg/resource.h

```diff
@@ -20,6 +20,12 @@
 #include "meta.h"
 #include <workerd/jsg/modules.capnp.h>
 
+// The signature of SetAccessor changes in v8 12.1 to drop the v8::AccessControl
+// parameter.
+#if (V8_MAJOR_VERSION <= 12) && (V8_MINOR_VERSION < 1)
```

Comment:
I guess it doesn't actually matter to us, but technically this should be:

```
#if (V8_MAJOR_VERSION < 12) || ((V8_MAJOR_VERSION == 12) && (V8_MINOR_VERSION < 1))
```

Otherwise it does the wrong thing for e.g. version 11.9.